.github/workflows/ci.yml

```diff
@@ -59,6 +59,9 @@ jobs:
       - name: nvm use
         run: nvm use
 
+      - name: Set Ruby version
+        uses: ruby/setup-ruby@v1 # Sets ruby version via `.ruby-version`
```

Comment:
Is this official or do we need to check its legitimacy?

Reply:
It's from the ruby org so pretty legit I reckon 🤔